### PR TITLE
[Network] `az network watcher connection-monitor create`: Add VMSS support in CM source/destination

### PR DESCRIPTION
**Pull request by kumaam**
**Related command**

az network watcher connection-monitor create

**Description**

Azure VMSS support is now available in connection-monitor.  Support for VMSS as connection-monitor endpoints is now availbale. This change will enable customer to create connection-monitor with VMSS as endpoints via Azure CLI.

New parameters are added -

--endpoint-source-address-include- Space-separated list of IP addresses to include in CM endpoint
--endpoint-source-address-exclude- Space-separated list of IP addresses to exclude in CM endpoint

**Testing Guide**
- Run the command to verify Connection monitor is getting created with VMSS as source/destination- 
az network watcher connection-monitor create -n cmVMSSCli --endpoint-source-name vmssSource --endpoint-source-type AzureVMSS --endpoint-source-resource-id /subscriptions/**************************/resourceGroups/rg-eastus2euap/providers/Microsoft.Compute/virtualMachineScaleSets/vmss-eastus2euap --endpoint-source-address-include 10.69.0.4 --endpoint-source-address-exclude 10.69.0.6 --endpoint-dest-name vmssdest --endpoint-dest-resource-id /subscriptions/**********************/resourceGroups/rg/providers/Microsoft.Compute/virtualMachineScaleSets/vmss --endpoint-dest-address-include 10.128.0.4 --endpoint-dest-address-exclude 10.128.0.5 --endpoint-dest-type AzureVMSS --test-config-name tcptc --protocol Tcp --tcp-port 2048

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Network

**Review by necusjz — APPROVED** [review]

File: linter_exclusions.yml [hunk 1]
```diff
@@ -2818,12 +2818,24 @@ network watcher connection-monitor create:
     endpoint_dest_resource_id:
       rule_exclusions:
       - option_length_too_long
+    endpoint_dest_address_include:
```
Comment (necusjz) [inline linter_exclusions.yml:2821]: Better not to modify "linter_exclusions" and use "options_list" instead, e.g.,
https://github.com/Azure/azure-cli/blob/efdecf4142ce23cccfe2169227b806201b87d0b2/src/azure-cli/azure/cli/command_modules/network/_params.py#L500-L502

Comment (kumaam) [reply linter_exclusions.yml:2821]: To make this parameter name meaningful and consistent with other parameters, it has to be added in linter_exclusions.
All source endpoints start with - `endpoint_source*` and destination endpoints starts with `endpoint_dest*`. 

Hence, we'd added it in linter_exclusions, so the argument names would be meaningful and not confusing. Let me know if you see any concerns.

**Comment by kumaam** [comment]
Discarding this PR, as we need to support mult-endpoints in CM (i.e multiple sources/destination in a CM).


Will reopen a PR with all those changes.

**Pull request closed without merging**